Example/WalletApp/ApplicationLayer/SceneDelegate.swift

```diff
@@ -31,6 +34,7 @@ final class SceneDelegate: UIResponder, UIWindowSceneDelegate {
         app.uri = connectionOptions.urlContexts.first?.url.absoluteString.replacingOccurrences(of: "walletapp://wc?uri=", with: "")
 
         configurators.configure()
+        pushRegisterer.registerForPushNotifications()
```

Comment:
We can put PushRegisterer into out service container and call:
`app.pushRegisterer.registerForPushNotifications()`